src/main/java/interface_adapter/progresstracker/ProgressTrackerController.java

```diff
@@ -0,0 +1,19 @@
+package interface_adapter.progresstracker;
+
+import entity.Media;
```

Comment:
importing entity into controller violates Clean Architecture. can get logged in user from the view model. The Media data can be loaded in the interactor instead I think.